plugins/modules/mssql_script.py

```diff
@@ -325,8 +356,17 @@ def run_module():
             msg="when supplying login_user argument, login_password must also be provided")
 
     try:
-        conn = pymssql.connect(
-            user=login_user, password=login_password, host=login_querystring, database=db)
+        connection_args = {
+            "user": login_user,
+            "password": login_password,
+            "host": login_querystring,
+            "database": db,
+        }
+        if encryption is not None:
+            connection_args["encryption"] = encryption
```

Comment:
if you make the parameter a boolean, then this should go:
```suggestion
            connection_args["encryption"] = "on" if encryption else "off"
```